league/oauth2-server-bundle/0.4/post-install.txt

```diff
@@ -0,0 +1,10 @@
+  1. <bg=magenta;fg=white> Provide a key pair </>
+    * Generate a private/public key pair (with passphrase) at the locations defined in your .env file: <comment>https://oauth2.thephpleague.com/installation/#generating-public-and-private-keys</>
+    * Configure the <info>private_key</> and <info>public_key</> with the respective key locations
+    * Set the <info>private_key_passphrase</> to the private key password set in the previous step
```

Comment:
In practice, they should set the `OAUTH_PASSPHRASE` env var, right? If so, does it make sense to "generate" one for them automatically in the recipe?

Reply:
Actually it's already the case, the post-install suggestion was wrong here. Updated